src/background.js

```diff
@@ -57,7 +57,7 @@ const stateManager = {
     }
 
     let prevMode = await rollout.getSetting("doh-rollout.previous.trr.mode", 0);
-    let curMode = await browser.experiments.preferences.getUserPref(
+    let curMode = await browser.experiments.preferences.getIntPref(
```

Comment:
I was surprised to see this, since we had only talked about adjusting the `doh-rollout-enabled` pref. After looking around a bit, I think this will be ok to keep. I can't think of a way this would work other than intended.